cmake/GroongaSphinx.cmake

```
@@ -74,8 +74,9 @@ function(grn_sphinx SOURCE_DIR LOCALE SOURCES HTML_FILES)
       COMMAND
         ${CMAKE_COMMAND} -E env DOCUMENT_VERSION=${GRN_VERSION}
         DOCUMENT_VERSION_FULL=${GRN_VERSION_FULL} LOCALE=${LOCALE}
-        ${SPHINX_BUILD} -j auto -D language=${LOCALE} -b ${BUILDER} -d
-        ${LOCALE}/doctrees/${BUILDER} ${SOURCE_DIR} ${LOCALE}/${BUILDER}
+        ${SPHINX_BUILD} ${SPHINX_BUILD_OPTS} -j auto -D language=${LOCALE} -b
```

デバッグビルドのときはデフォルトで有効にしちゃって。

```suggestion
        ${SPHINX_BUILD} $<$<CONFIG:Debug>:--fail-on-warning> -j auto -D language=${LOCALE} -b
```